File: Logica JS/log1.js
let N = parseInt(prompt("-- Digite um valor N inteiro --"));

for (let i = 0; i < N; i++) {
  let valor = parseInt(prompt(`Digite o ${i + 1}º valor:`));

  if (valor % 2 === 0) {
    console.log("-> É um número par.");
  } else {
    console.log("-> É um número ímpar.");
  }
}

console.log("Programa encerrado.");

File: Logica C/log1.c
#include <stdio.h>

int main(void) {
  int valor, i, N;
  printf("-- Digite um N valor inteiro -- \n");
  scanf("%d" , &N);

  for (i = 0; i < N; i++) {
    printf("Digite o %do valor: ", i + 1);
    scanf("%d", &valor);

    if (valor % 2 == 0) {
      printf("-> É um número par.\n");
    } else {
      printf("-> É um número ímpar.\n");
    }
  }

  return 0;
}